dftimewolf/lib/processors/gcp_logging_timesketch.py

```diff
@@ -16,7 +16,47 @@
 
 
 class GCPLoggingTimesketch(BaseModule):
-  """Transforms Google Cloud Platform logs for Timesketch."""
+  """Transforms Google Cloud Platform logs for Timesketch.
+
+  Attributes:
+    data_type (str): Timesketch data type i.e. gcp:log:json.
+    datetime (str): event date time.
+    dcsa_emails (List[str]): default compute service account, a service account
+        attached when a Compute Engine instance is created.
+    dcsa_scopes (List[str]): OAuth scopes granted to the default service account
+        attached to a Compute Engine instance.
+    delegation_chain (str): service account impersonation/delegation chain.
+    event_subtype (str): event subtype.
+    gcloud_command_id (str): unique gcloud command execution ID.
+    gcloud_command_partial (str): partial gcloud command related to the
+        operation.
+    message (str): summary message of the operation.
+    method_name (str): operation performed.
+    permissions (List[str]): IAM permissions used for the operation.
+    policy_delta (List[str]): IAM policy delta.
+    principal_email (str): email address of the requester.
+    principal_subject (str): subject of the requester.
+    query (str): Google Cloud log filtering query.
+    resource_label_instance_id (str): Compute Engine instance ID.
+    resource_name (str): resource name.
+    service_account_delegation (list[str]): service accounts delegation in
+        authentication.
+    service_account_display_name (str): display name of the service account.
+    service_account_key_name (str): service account key name used in
+        authentication.
+    service_name (str): name of the service.
+    severity (str): log entry severity.
+    source_images (List[str]): source images of disks attached to a Compute
+        Engine instance.
+    source_ranges (List[str]): firewall source ranges.
+    status_code (str): operation success or failure code.
+    status_message (str): operation success or failure message.
+    status_reason (str): operation failure reasons.
+    textPayload (str): text payload for logs not using a JSON or proto payload.
+    timestamp_desc (str): description of timestamp.
+    user (str): user or requestor.
+    user_agent (str): user agent used in the request.
+  """
```

Comment:
The `Attributes` list in a class docstring usually refers to the members of the class, but this appears to be log fields in the output. Maybe it would be better to list these as a file level comment/docstring?

Reply:
Changed to file docstring.